src/System.Device.Gpio/System/Device/Media/PixelFormat.cs

```diff
@@ -0,0 +1,463 @@
+        SBGGR16 = 844257602,
+        SGBRG16 = 909197895,
+        SGRBG16 = 909201991,
+        SRGGB16 = 909199186,
+        HSV24 = 861295432,
+        HSV32 = 878072648,
+        MJPEG = 1196444237,
+        JPEG = 1195724874,
+        DV = 1685288548,
+        MPEG = 1195724877,
+        H264 = 875967048,
+        H264_NO_SC = 826496577,
+        H264_MVC = 875967053,
+        H263 = 859189832,
+        MPEG1 = 826757197,
+        MPEG2 = 843534413,
+        MPEG2_SLICE = 1395803981,
+        MPEG4 = 877088845,
+        XVID = 1145656920,
+        VC1_ANNEX_G = 1194410838,
+        VC1_ANNEX_L = 1278296918,
+        VP8 = 808996950,
+        VP9 = 809062486,
+        HEVC = 1129727304,
+        FWHT = 1414027078,
+        FWHT_STATELESS = 1213679187,
+        CPIA1 = 1095323715,
+        WNVA = 1096175191,
+        SN9C10X = 808532307,
+        SN9C20X_I420 = 808597843,
+        PWC1 = 826496848,
+        PWC2 = 843274064,
+        ET61X251 = 892483141,
+        SPCA501 = 825242963,
+        SPCA505 = 892351827,
+        SPCA508 = 942683475,
+        SPCA561 = 825636179,
+        PAC207 = 925905488,
+        MR97310A = 808530765,
+        JL2005BCD = 808602698,
+        SN9C2028 = 1481527123,
+        SQ905C = 1127559225,
+        PJPG = 1196444240,
+        OV511 = 825308495,
+        OV518 = 942749007,
+        STV0680 = 808990291,
+        TM6000 = 808865108,
+        CIT_YYVYUY = 1448364355,
+        KONICA420 = 1229868875,
+        JPGL = 1279742026,
+        SE401 = 825242707,
+        S5C_UYVY_JPG = 1229141331,
+        Y8I = 541669465,
+        Y12I = 1228026201,
+        Z16 = 540422490,
+        MT21C = 825381965,
+        INZI = 1230655049,
+        SUNXI_TILED_NV12 = 842093651,
+        CNF4 = 877022787,
+        IPU3_SBGGR10 = 1647538281,
+        IPU3_SGBRG10 = 1731424361,
+        IPU3_SGRBG10 = 1194553449,
+        IPU3_SRGGB10 = 1915973737,
+        CU8 = 942691651,
+        CU16LE = 909202755,
+        CS8 = 942691139,
+        CS14LE = 875647811,
+        RU12LE = 842093906,
+        PCU16BE = 909198160,
+        PCU18BE = 942752592,
+        PCU20BE = 808600400,
+        DELTA_TD16 = 909198420,
+        DELTA_TD08 = 942687316,
+        TU16 = 909202772,
+        TU08 = 942691668,
+        VSP1_HGO = 1213223766,
+        VSP1_HGT = 1414550358,
+        UVC = 1212372565,
+        D4XX = 1482175556,
+    }
+
+    /// <summary>
+    /// videodev2.h Pixel Format Definition
+    /// </summary>
+    internal class RawPixelFormat
+    {
+        public static void PrintFields()
+        {
+            Type type = typeof(RawPixelFormat);
+
+            var fields = type.GetFields(BindingFlags.Static | BindingFlags.Public);
+            foreach (var field in fields)
+            {
+                Console.WriteLine($"{field.Name} = {field.GetValue(new RawPixelFormat())},");
+            }
+        }
+
+        public static uint v4l2_fourcc(uint a, uint b, uint c, uint d) => a | (b << 8) | (c << 16) | (d << 24);
```

Comment:
if you're passing this to native you might want to consider fixed size buffer of bytes for this